package.json

```diff
@@ -29,41 +29,40 @@
   },
   "dependencies": {
     "@openapi-contrib/openapi-schema-to-json-schema": "^3.0",
-    "@stoplight/json": "^3.4.0",
+    "@stoplight/json": "^3.5.1",
     "@stoplight/types": "^11.4.0",
     "@types/swagger-schema-official": "~2.0.20",
     "@types/to-json-schema": "^0.2.0",
     "@types/type-is": "^1.6.3",
-    "@types/urijs": "~1.19.4",
+    "@types/urijs": "~1.19.7",
     "lodash": "^4.17.15",
     "openapi3-ts": "~1.3.0",
-    "postman-collection": "^3.5.5",
+    "postman-collection": "^3.6.0",
     "to-json-schema": "^0.2.3",
     "type-is": "^1.6.18",
     "urijs": "~1.19.2"
   },
   "devDependencies": {
-    "@stoplight/scripts": "^5",
+    "@stoplight/scripts": "^8.2.0",
     "@stoplight/test-utils": "^0.0.1",
-    "@types/jest": "24.0.25",
-    "@types/json-schema": "7.0.x",
+    "@types/jest": "25.1.4",
+    "@types/json-schema": "7.0.4",
     "@types/lodash": "4.14.149",
-    "@types/postman-collection": "^3.5.0",
-    "jest": "24.9.0",
+    "@types/postman-collection": "^3.5.1",
+    "jest": "25.1.0",
     "prettier": "1.19.1",
-    "ts-jest": "24.3.0",
-    "ts-node": "^8.6.0",
-    "tslint": "5.20.1",
+    "ts-jest": "25.2.1",
+    "ts-node": "^8.6.2",
+    "tslint": "6.1.0",
```

Comment:
Should we bit the bullet and make the switch over to our newer eslint approach? https://github.com/stoplightio/eslint-config#usage

Reply:
I vote to not use that preset yet, it's asking for too many dependencies we just do not need here (react hooks and stuff)